src/__tests__/pagopa.test.ts

```diff
@@ -133,25 +133,92 @@ describe("PaymentNoticeNumberFromString", () => {
 });
 
 describe("QrCodeFromString", () => {
-  it("should succeed with valid QrCode", async () => {
-    const qrCodeSrt = "PAGOPA|002|123456789012345678|12345678901|1234567801";
-    const validation = PaymentNoticeQrCodeFromString.decode(qrCodeSrt);
-    expect(isRight(validation)).toBeTruthy();
-    if (isRight(validation)) {
-      expect(validation.value.amount).toHaveLength(10);
-      expect(validation.value.identifier).toHaveLength(6);
-      expect(validation.value.version).toHaveLength(3);
-      expect(validation.value.organizationFiscalCode).toHaveLength(11);
-      expect(validation.value.paymentNoticeNumber.auxDigit).toEqual("1");
-      // tslint:disable-next-line:no-any
-      expect((validation.value.paymentNoticeNumber as any).iuv17).toEqual(
-        "23456789012345678"
-      );
-      expect(PaymentNoticeQrCodeFromString.encode(validation.value)).toEqual(
-        qrCodeSrt
-      );
+  it.each([
+    [
```

Comment:
The tests fail by adding this test

```
[
      "PAGOPA|002|223456789012345678|12345678901|1234567801",
      "234567890123456",
      10,
      "2",
      "should succeed with valid QrCode"
    ]
```
Could we consider to test also the `aux 2` case?

```typescript
else if (expectedAuxDigit === "2") {
          expect(
            (validation.value.paymentNoticeNumber as PaymentNoticeNumber2).iuv15
          ).toEqual(paymentNoticeNumber);
        }
```

Reply:
Fixed in 3273538